package.json

```diff
@@ -56,6 +56,7 @@
     "webpack": "^1.13.1"
   },
   "dependencies": {
-    "classnames": "^2.2.5"
+    "classnames": "^2.2.5",
+    "react-bootstrap": "^0.30.3"
```

Comment:
I think we should move this out of dependencies and into `peerDependencies` so we're not forcing a certain version on the end user.